.travis.yml

```diff
@@ -31,15 +31,21 @@ install:
       bash $MINICONDA_FILE -b
 
       export PATH=/Users/travis/miniconda3/bin:$PATH
-
-      conda config --set show_channel_urls true
-      conda update --yes conda
-      conda install --yes conda-build=1.20.0 jinja2 anaconda-client
       conda config --add channels conda-forge
-      
+      conda config --set show_channel_urls true
+      conda install --yes --quiet conda-forge-build-setup
+      source run_conda_forge_build_setup
 
 script:
   - conda build ./recipe
+  # Test libpng pinnings
+  - conda create -n temp01 --override-channels -c conda-forge -c defaults python=2.7 matplotlib --yes
+  - source activate temp01
+  - python -c 'import matplotlib.pyplot'
+  - conda install --no-update-deps libpng=1.6.21 --yes
+  - python -c 'import matplotlib.pyplot'
+  - conda install --no-update-deps libpng=1.6.23 --yes
+  - python -c 'import matplotlib.pyplot'
```

Comment:
Not sure what you mean.

The test is to check if `matplotlib` compiled with `libpng 1.6.24` works when another `libpng 1.6.21` or `1.6.23` is installed.

I cannot reproduce #41 on Linux and, since I don't have a Mac, I am using this PR to test. Hopefully we cando a better pinning and fix @183amir's problems.